src/containers/my-resources/add-resource-with-input/AddResourceWithInput.tsx

```diff
@@ -23,14 +27,20 @@ interface AddResourceWithInputProps {
   link?: string
   searchRef: MutableRefObject<string>
   button?: ReactElement
+  categoryService?: ServiceFunction<string[], undefined>
```

Comment:
why you pass the same service in every table container if you can remove this prop and pass getResourcesCategoriesNames directly to FilterSelector component

Reply:
change a bit

Reply:
but you still pass it as a prop